src/index.js

```diff
@@ -2,6 +2,15 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 
+class Try extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render (){
+    return(  <h2>TRY: {this.props.count}</h2>)
```

Comment:
`return(  <h2>**Try #**: {this.props.count}</h2>)`

Reply:
both hashtag and column?